src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,29 @@
+package duke.command;
+
+import duke.util.TaskList;
+
+/**
+ * Class that represents the Command to delete from the TaskList.
+ *
+ * @author Benedict Chua
+ */
+public class DeleteCommand extends Command {
+    private TaskList tasks;
+    private int index;
+
+    /**
+     * Constructor for DeleteCommand.
+     *
+     * @param tasks TaskList containing current tasks.
+     * @param index The index pointing to the Task in TaskList to operate on, starting from 1.
+     */
+    public DeleteCommand(TaskList tasks, int index) {
+        this.tasks = tasks;
+        this.index = index;
+    }
+
+    @Override
```

Comment:
Can put @inheritDocs here, just a nit :grin:

Reply:
Thanks for the suggestion! Did not that existed 😄